app/views/shared/check_answers/_dependants.html.erb

```diff
@@ -1,66 +1,70 @@
 <% if @legal_aid_application.dependants %>
-  <%- @legal_aid_application.dependants.each_with_index do |dependant, index| %>
-    <div class="govuk-grid-row" id="app-check-your-answers__dependants_<%= index + 1 %>_delete">
-      <div class="govuk-grid-column-one-third">
+  <% @legal_aid_application.dependants.each_with_index do |dependant, index| %>
+    <div class="govuk-grid-row" id="app-check-your-answers__dependants_<%= index + 1 %>">
+      <div class="govuk-grid-column-two-thirds">
         <h2 class="govuk-heading-m">Dependant <%= index + 1 %></h2>
       </div>
+      <div class="govuk-grid-column-one-third govuk-summary-list--no-border align-text-right">
+        <p>
+          <%= link_to(providers_legal_aid_application_dependant_path(@legal_aid_application, dependant), class: 'govuk-link') do %>
+            <%= t('generic.change') %><span class="govuk-visually-hidden">Change dependant <%= dependant.name %></span>
+          <% end %>
+        </p>
+      </div>
     </div>
-
     <dl class="govuk-summary-list">
-
       <%= check_answer_link(
               name: :dependant,
-              url: nil,
-              question: t('providers.means_summaries.show.dependants.name'),
+              question:  t('providers.means_summaries.show.dependants.name'),
               answer: dependant.name,
               read_only: true,
-              align_right: false
+              align_right: true
           ) %>
       <%= check_answer_link(
               name: :dependant,
-              url: nil,
               question: t('providers.means_summaries.show.dependants.date_of_birth'),
               answer: dependant.date_of_birth,
               read_only: true,
-              align_right: false
+              align_right: true
           ) %>
       <%= check_answer_link(
               name: :dependant,
-              url: check_answer_url_for(:providers, :dependants_relationships, @legal_aid_application, dependant_id: dependant),
+              url: nil,
```

Comment:
All looks good to me. Two clean up comments below:

1. Instead of having all the 'nil' values for the urls, could you use the #check_answer_no_link from the CheckAnswersHelper.

2. Also, on line 62 there's a comment "<!-- pasted here -->" i'm assuming that's no longer needed

Reply:
Cheers @Obsiye I have removed the comment. You were correct, it was no longer needed! 😄 
I have completely re-written the view, it was always in the back of my mind but I got distracted, could you have a look at the subsequent commits and let me have your feedback?